renku/core/commands/login.py

```diff
@@ -80,14 +100,40 @@ def _get_url(parsed_endpoint, path, **query_args):
     return parsed_endpoint._replace(path=path, query=query).geturl()
 
 
-def _store_token(client, parsed_endpoint, token):
-    client.set_value(section=CONFIG_SECTION, key=parsed_endpoint.netloc, value=token, global_only=True)
+def _store_token(client, parsed_endpoint, access_token):
+    client.set_value(section=CONFIG_SECTION, key=parsed_endpoint.netloc, value=access_token, global_only=True)
     os.chmod(client.global_config_path, 0o600)
 
 
+def _store_git_token(client, access_token):
+    # NOTE: Do not use "Authorization: Bearer token" because it interferes with Git LFS authentication
+    header = f"Renku-Auth-Access-Token: {access_token}"
+    client.repo.git.config("http.extraheader", header, local=True)
+
+
+def _swap_git_remote(client, parsed_endpoint, remote_name, remote_url):
+    new_remote_url = get_renku_repo_url(remote_url, deployment_hostname=parsed_endpoint.netloc)
+
+    backup_remote_name = f"{RENKU_BACKUP_PREFIX}-{remote_name}"
+    try:
+        client.repo.create_remote(backup_remote_name, url=remote_url)
+    except git.GitCommandError:
+        communication.error(f"Cannot create backup remote '{backup_remote_name}' for '{remote_url}'")
+    else:
+        try:
+            client.repo.git.remote("set-url", remote_name, new_remote_url)
+        except git.GitCommandError:
+            raise errors.GitError(f"Cannot change remote url for '{remote_name}' to {new_remote_url}")
```

Comment:
Should we remove the backup remote if this failed? Otherwise the command might fail if a user retries and the backup remote already exists?